src/stateless/icon.vue

```diff
@@ -4,6 +4,13 @@
             <div class="icon-caret"></div>
         </div>
 
+        <svg v-if="name === 'schedule-clock'" :style="styleOverride" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+            <g fill-rule="nonzero">
```

Comment:
When exporting SVG from Sketch, there are always some unneeded residuals like fill-rule="nonzero". To keep SVGs clean you should use [SVGomg](https://jakearchibald.github.io/svgomg/).

Reply:
This went through svgomg but it doesn't remove fill-rule. It is redundant in this case so I manually removed it.